scripts/linters/js_ts_linter.py

```diff
@@ -1062,7 +1062,7 @@ def perform_all_lint_checks(self):
             return []
 
         # Clear temp compiled typescipt files from previous runs.
-        clean.delete_directory_tree(COMPILED_TYPESCRIPT_TMP_PATH)
+        shutil.rmtree(COMPILED_TYPESCRIPT_TMP_PATH, ignore_errors=True)
```

Comment:
Why this change?

Reply:
Because tmp directory was not being deleted before if that has some files in it.

Reply:
Then change the `clean.delete_directory_tree` if that's not working as expected?

Reply:
That file is being used in many places, even we have this same code before David changed the structure.